api/src/test/java/org/openmrs/notification/db/hibernate/HibernateAlertDAOTest.java

```diff
@@ -9,34 +9,63 @@
  */
 package org.openmrs.notification.db.hibernate;
 
+import org.junit.Assert;
 import org.junit.Before;
 import org.junit.Test;
+import org.openmrs.User;
 import org.openmrs.api.context.Context;
-import org.openmrs.notification.AlertService;
+import org.openmrs.notification.Alert;
 import org.openmrs.test.BaseContextSensitiveTest;
+import org.springframework.beans.factory.annotation.Autowired;
 
 /**
  * This class tests the hibernate alert data access. TODO Consider changing this and all subsequent
  * tests to use dbunit
  */
 public class HibernateAlertDAOTest extends BaseContextSensitiveTest {
 	
+	private static final String DATA_XML = "org/openmrs/api/db/hibernate/include/HibernateAlertDAOTestDataSet.xml";
+	
+	@Autowired
+	private HibernateAlertDAO hibernateAlertDAO;
+	
 	@Before
-	public void runBeforeEachTest() {
-		authenticate();
+	public void setUp() {
+		executeDataSet(DATA_XML);
+	}
+
+	@Test
+	public void saveAlert_shouldSaveAlertToDb() {
+		Alert saveAlert = hibernateAlertDAO.getAlert(2);
+	    hibernateAlertDAO.saveAlert(saveAlert);
+	    Assert.assertNotNull(saveAlert.getText());
```

Comment:
I'm not sure what, exactly, is being tested here. Presumably, we should create a new alert (rather than loading one from the database), save it, and assert that we can call `getAlert()` to get the expected alert.